ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/migrations/V3__Backfill_Key_Hash_State_Table.scala

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.on.sql.migrations
+
+import anorm.{BatchSql, NamedParameter}
+import com.daml.ledger.on.sql.queries.StateKeyHashing
```

Comment:
Consider inlining the code from this import. Migrations should be immutable, if we ever change `StateKeyHashing`, we will silently break this migration. We had many issues with the `V2_1__Rebuild_Acs` migration of the index database for this reason.

Reply:
That's a good point. Thanks!

Reply:
Fixed.